roles/common/tasks/redhat.yml

```diff
@@ -102,5 +114,5 @@
 - name: Enable FIPS on rhel 8+
   include_tasks: fips-redhat.yml
   when:
-    - ansible_os_family == "RedHat" and ansible_distribution_major_version in ['8', '9']
+    - ansible_os_family == "Redhat" and ansible_distribution_major_version in ['8', '9', '2023']
```

Comment:
For more clarity this should look something like (conditions for rhel 8,9 ) or (conditions for amazon linux) rather than just adding the ansible_distribution_major_version for amazon.